src/main/java/me/botsko/prism/actions/BlockAction.java

```diff
@@ -410,15 +411,15 @@ protected ChangeResult placeBlock(Player player, QueryParameters parameters, boo
                 final Skull skull = (Skull) block.getState();
                 skull.setRotation( s.getRotation() );
                 skull.setSkullType( s.getSkullType() );
-                if( s.owner != null && !s.owner.isEmpty() ) {
+                if( !Strings.isNullOrEmpty(s.owner) ) {
```

Comment:
I disagree; I'd do the same thing. Subjectively speaking, I find it cleaner to read and understand than two separate operations. But not a big deal for me really :P